governance/default.go

```diff
@@ -1194,3 +1150,54 @@ func (gov *Governance) IdxCacheFromDb() []uint64 {
 	res, _ := gov.db.ReadRecentGovernanceIdx(0)
 	return res
 }
+
+// Returns the istanbul epoch. This function works even before loading any params
+// from database. We need epoch to load any param from database.
+func (gov *Governance) epochWithFallback() uint64 {
+	// Comes here in most cases
+	if v, ok := gov.Params().Get(params.Epoch); ok {
+		return v.(uint64)
+	}
+	// Useful before first UpdateParam()
```

Comment:
Useful..? Does it mean `below if loop is used before first UpdateParam()`? What about changing to more specific sentence?

Reply:
Fixed in https://github.com/klaytn/klaytn/pull/1424/commits/3e859a5b401be6e4d813590a6d7921526a4d59a3.

Reply:
Thanks, @blukat29